google/ggrc-core
# GGRC-5921 Make Assertions mandatory for Control

# Issue description

Now `assertions` are mandatory for `Control` on FE only. They needed to be made mandatory on BE too. Also migration for already existing controls without assertions needed to set default value for assertions (which is **Security**) and create proper revisions.

# Steps to test the changes

1. Try to create `Control` with curl or any other REST client;
2. Check response status , it should be 400;
3. Check database, no `Control` should be created.

# Solution description

Proposed solution includes creating `hybrid_property` for `assertions` on `Control` model with custom setter method which checks whether provided values are valid. If not, `ValueError` will be raised.

Also PR includes migration which creates mappings between **Security** category and controls without any assertions.

# Sanity checklist


# Migration checklist


# PR Review checklist
